### PR TITLE
extraction of resourceful actions into separate component

### PR DESCRIPTION
**Pull request by notks**

### CODE REVIEW

File: src/components/ResourcefulActions.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+type Props = {
+  resource: string;
+  actions: Actions[];
+};
+
+export default function RecorcefulActions(props: Props) {
+  const [t] = useTranslation();
+  return (
+    <Dropdown label={t('actions')}>
+      {props.actions.map((action) => {
```
Comment (beganovich) [inline src/components/ResourcefulActions.tsx:28]: As far as I know, this should throw a warning in both console & editor with missing key prop?

**Review by beganovich — COMMENTED** [review]
Looks good, slight change.

File: src/components/ResourcefulActions.tsx (hunk 1 above)
Comment (notks) [reply src/components/ResourcefulActions.tsx]: I did that just forgot to add it to pr. It should be good now.

**Comment by beganovich** [comment]
Thanks for your contribution! :+1:

**Pull request merged**